packages/pangea-sdk/pangea/request.py

```diff
@@ -49,12 +50,25 @@ def __init__(
 
         # Custom headers
         self._extra_headers = {}
-
         self.session: requests.Session = self._init_session()
 
+        self.logger = logging.getLogger(self.service)
```

Comment:
i dont think we should do this automatically. if the user enables us to do our own logging or provides a logger to us then we should log. if the user provides us a logger we should not set our own format or our own handler. the logs need to go to wherever the customer has set up logging to sink to and in whatever format the customer is expecting to see. we can have a default one that the customer can enable, but it shouldnt be on by default

Reply:
agree. all we should just do is `self.logger = logging.getLogger(self.service)` and nothing else, we can have an override for them to set a logger name instead of self.service

Reply:
@govind-salinas-pangea @doubletooth Thanks for feedback. I've done some changes in base of your comments. Let me know what you think!